apps/api/src/domain/models/audio-visual/video/repositories/arango-video-query-repository.ts

```diff
@@ -34,6 +39,14 @@ export class ArangoVideoQueryRepository implements IVideoQueryRepository {
         );
     }
 
+    async createNoteAbout(
+        id: string,
+        noteId: string,
```

Comment:
After some reflection, I think we should inject the non-id props from the event payload here and do away with the database join.

Reply:
This has been done now.